operator/src/main/resources/scripts/startNodeManager.sh

```diff
@@ -309,11 +312,11 @@ while [ 1 -eq 1 ]; do
     break
   fi
   if [ $((SECONDS - $start_secs)) -ge $max_wait_secs ]; then
-    trace "Info: Contents of node manager log '$nodemgr_log_file':"
+    trace INFO "Contents of node manager log '$nodemgr_log_file':"
```

Comment:
@rjeberhard @alai8 @lennyphan Should I make lines 315 & 317 SEVERE instead of INFO?  

My thinking was that the SEVERE at line 319 was sufficient to get folks to dig deeper into the pod log and fine the INFO messages...

Reply:
(The INFO messages will contain the output from the following 'cat' when they are parsed and added to the operator log.)

Reply:
Agree. I think the SEVERE at line 319 should be sufficient.